src/NuGet.Core/NuGet.Protocol/HttpSource/HttpSource.cs

```diff
@@ -302,7 +310,7 @@ public async Task<JObject> GetJObjectAsync(HttpSourceRequest request, ILogger lo
             HttpResponseMessage response;
             try
             {
-                response = await RetryHandler.SendAsync(request, log, cancellationToken);
+                response = await RetryHandler.SendAsync(request, _packageSource.Name, log, cancellationToken);
```

Comment:
Shouldn't it be source/
packageSource is the name of the source itself in the nuget.config

Reply:
it's possible for the same source url to be listed twice, under different names. I think it would be useful to measure the two separately. My understanding is the source name (key in the config file) is always unique.

Reply:
> it's possible for the same source url to be listed twice,

That's deduped early though. At least in PackageReference it is.

Reply:
If they are not, they sure as hell should be :)
This would allow us to correlate performance of something like nuget.org without worrying about the name of the source.

Reply:
If you check my other PR, I look up the sources by name to find the URL, and use that when reporting. So, we'll be able to correlate sources, regardless of source name. But I don't feel strongly about this, and it makes reporting easier since I'll no longer need to get the list of sources.

Reply:
yeah, I'd say just use the value. 
It doesn't have to be this PR though.